src/ProviderHub/examples/Get-AzProviderHubCustomRollout.md

```diff
@@ -0,0 +1,6 @@
+### Example 1: Get a custom rollout by rollout name.
```

Comment:
There are no output for all of the examples. Please add them by following an example like https://github.com/Azure/azure-powershell/blob/generation/src/Databricks/examples/Get-AzDatabricksWorkspace.md.